src/WooCommerce.php

```diff
@@ -378,7 +378,7 @@ public static function getOrderDetailsHtmlDataAttr($order_id) {
     $order_data = $order->get_data();
 
     $order_details = [
-      'id' => $order->get_order_number(),
+      'order_number' => $order->get_order_number(),
```

Comment:
As this maps to the predefined EECpurchase event data object, I would recommend to not change the key name.
```suggestion
      'id' => $order->get_order_number(),
```